test/e2e/grpc/client.test.ts

```diff
@@ -318,4 +319,26 @@ describe('grpc/client', () => {
       await stream2.stop();
     });
   });
-});
+
+  describe('distributed lock', () => {
```

Comment:
Some more scenarios:
1. two owners are acquiring the same lock at the same time 
2. two owners are acquiring different locks on unique resourceIDs 
3. unlocking a lock that belongs to someone else